checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -231,10 +240,10 @@ public boolean isSubtype(AnnotationMirror subAnno, AnnotationMirror superAnno) {
                 return AnnotationUtils.areSameByClass(superAnno, PolySameLen.class);
             } else if (AnnotationUtils.hasElementValue(subAnno, "value")
                     && AnnotationUtils.hasElementValue(superAnno, "value")) {
-                List<String> a1Val = getValueOfAnnotationWithStringArgument(subAnno);
-                List<String> a2Val = getValueOfAnnotationWithStringArgument(superAnno);
+                List<String> subArrays = getValueOfAnnotationWithStringArgument(subAnno);
+                List<String> superArrays = getValueOfAnnotationWithStringArgument(superAnno);
 
-                if (overlap(a1Val, a2Val)) {
+                if (subArrays.containsAll(superArrays)) {
```

Comment:
The code says that `@SameLen({"a", "b"})` is a subtype of `@SameLen("a")`, so I think the image is wrong.